packages/gatsby/src/redux/actions/restricted.js

```diff
@@ -258,12 +329,15 @@ const mapAvailableActionsToAPIs = restrictions => {
 }
 
 const availableActionsByAPI = mapAvailableActionsToAPIs({
+  createFieldExtension: {
+    [ALLOWED_IN]: [`sourceNodes`, `createSchemaCustomization`],
+  },
   createTypes: {
-    [ALLOWED_IN]: [`sourceNodes`],
+    [ALLOWED_IN]: [`sourceNodes`, `createSchemaCustomization`],
```

Comment:
Should we make all those deprecated in `sourceNodes`?

Reply:
I kept `sourceNodes` as well because @pieh [comment here](https://github.com/gatsbyjs/gatsby/pull/14283#issuecomment-495664067)